lua/lang/svelte.lua

```diff
@@ -0,0 +1,35 @@
+local M = {}
+
+M.config = function()
+  O.lang.svelte = {}
+end
+
+M.format = function()
+  -- TODO: implement formatter (if applicable)
+  return "No formatter configured!"
+end
+
+M.lint = function()
+  -- TODO: implement linters (if applicable)
+  return "No linters configured!"
+end
+
+M.lsp = function()
+  if require("lv-utils").check_lsp_client_active "svelte" then
+    return
+  end
+
+  require("lspconfig").svelte.setup {
+    cmd = { DATA_PATH .. "/lspinstall/svelte/node_modules/.bin/svelteserver", "--stdio" },
+    filetypes = { "svelte" },
+    root_dir = require("lspconfig.util").root_pattern("package.json", ".git"),
```

Comment:
I think we should avoid overwriting the default values, lspconfig should be well enough configured, and also it creates bugs on our side because lspconfig get updates, but we keep outdated configs.

Reply:
aren't they the exact same?
I think if we want to do that, we need to do it for all `core/lang` files and avoid overwriting default values?
so do I create a new PR and change all of them, or is there something specifically wrong with this config?

Reply:
They are the same, but if lspconfig updates their defaults, we don't, you get that.
This needs to be done for all language servers, indeed, so I'll approve this in the meantime.
I started doing it today, but there is something I don't know how to do yet.
They apply the given user configuration on top of their default config, which is good, but this also means that it overwrites both the "cmd" entry and the options part. Also, I didn't manage to access their default config.

Reply:
I completely agree with you, even the maintainer of `nvim-lspconfig` was talking about this in discord

so i think we can ask him `mjlbach` in discord about that